python/graphstorm/model/utils.py

```diff
@@ -81,27 +88,43 @@ def sparse_emb_initializer(emb):
 def save_model(model_path, gnn_model=None, embed_layer=None, decoder=None):
     """ A model should have three parts:
         * GNN model
-        * embedding layer
+        * embedding layer, including a node encoder and an edge encoder. 
         The model is only used for inference.
 
-        Parameters
-        ----------
-        model_path: str
-            The path of the model is saved.
-        gnn_model: model
-            A (distributed) model of GNN
-        embed_layer: model
-            A (distributed) model of embedding layers.
-        decoder: model
-            A (distributed) model of decoder
+    .. versionchanged:: 0.5.1
+        Change the``embed_layer`` in v0.5.1 from a module into a dict with two elements to support
+        saving the edge encoder in a model.
+
+    Parameters
+    ----------
+    model_path: str
+        The path of the model is saved.
+    gnn_model: model
+        A (distributed) model of GNN
+    embed_layer: dict of model
```

Comment:
This is a breaking change.

Reply:
Why not just add an edge_embed_layer at the end of the arg-list? Then the code will be backward compatible.